pkg/operator/controller/status/controller.go

```diff
@@ -156,6 +165,26 @@ func New(mgr manager.Manager, config Config) (controller.Controller, error) {
 		})); err != nil {
 			return nil, err
 		}
+
+		isOSSMSubscription := predicate.NewPredicateFuncs(func(o client.Object) bool {
+			subscription, ok := o.(*operatorsv1alpha1.Subscription)
+			if !ok || subscription.Spec == nil {
+				return false
+			}
+			return ossmSubscriptions.Has(subscription.Spec.Package)
+		})
+
+		// The status controller needs to be aware of conflicting OLM subscriptions in any namespace. In order to
+		// prevent ballooning the main cache to watch all namespaces on all its informers, create a separate cache for
+		// tracking OLM subscriptions across all namespaces.
+		if reconciler.subscriptionCache, err = cache.New(mgr.GetConfig(), cache.Options{}); err != nil {
+			return nil, err
+		}
+		mgr.Add(reconciler.subscriptionCache)
```

Comment:
I expect this is a part of the initialization you need to do before calling `controller.New` in line 100.

Reply:
It works either way, but I moved this before `controller.New` for better readability